Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM centos:6
```

Comment:
I'm not sure about whether the folder approach is compatible with the automatic builds of the Docker-Hub or not. But even in this case, maybe we could store all the other files (config files, documentation, docker-compose files) in a dedicated folder instead.